### PR TITLE
Add SparkR tab

### PR DESCRIPTION
**Pull request by shivaram**
I've kept the regex to be 'SparkR' for now as 'R' might be too verbose. BTW does it ignore case by default ? (i.e. SparkR vs. SPARKR)

### CODE REVIEW

**Comment by JoshRosen** [comment]
Yep, it's case-insensitive matching: https://github.com/shivaram/spark-pr-dashboard/blob/128abefc61e6afc1c6c6b69efc75388a622a0ca8/sparkprs/models.py#L105

**Comment by shivaram** [comment]
Cool ! A way to test this would be to see if this PR https://github.com/apache/spark/pull/5436 shows up

File: sparkprs/models.py [hunk 1]
```diff
@@ -87,6 +87,7 @@ class Issue(ndb.Model):
         ("MLlib", "mllib", "mllib"),
         ("GraphX", "graphx|pregel", "graphx"),
         ("Streaming", "stream|flume|kafka|twitter|zeromq", "streaming"),
+        ("R", "SparkR", "r"),
```
Comment (JoshRosen) [inline sparkprs/models.py]: For the filename regex, we might want to only match files in the top-level `R` folder or in the Java half of the SparkR, so how about `(^r/)|src/main/r/|api/r/`, which I think should cover all R-related sources?

Comment (shivaram) [inline sparkprs/models.py]: Makes sense. Done

**Comment by JoshRosen** [comment]
Looks like it worked!  https://spark-prs.appspot.com/#r

**Pull request closed without merging**